evm/v0.5/package.json

```diff
@@ -1,16 +1,20 @@
 {
   "name": "chainlinkv0.5",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "license": "MIT",
+  "main": "./dist/src",
+  "typings": "./dist/src",
   "scripts": {
-    "build": "truffle build",
-    "build.windows": "truffle.cmd build",
+    "generate-typings": "typechain --target ethers --outDir src/generated \"{test/support/LinkToken*,dist/artifacts/*}\"",
+    "postgenerate-typings": "yarn workspace chainlink export-typings v0.5/src/generated v0.5/dist/src/generated",
+    "build:windows": "truffle.cmd build && yarn generate-typings && (tsc || true)",
+    "build": "sol-compiler && yarn generate-typings && (tsc || true)",
```

Comment:
Why mask `tsc` failures?

Reply:
Theres hundreds of tsc errors inside v0.5 since we havent converted it yet. Right now im just using it as a compiler with no typechecking